src/Microsoft.Identity.Client/AppConfig/AadAuthorityAudience.cs

```diff
@@ -45,10 +45,12 @@ namespace Microsoft.Identity.Client.AppConfig
     */
 
     /// <summary>
+    /// Specifies what Microsoft accounts can be signed-in in a given application.
     /// </summary>
     public enum AadAuthorityAudience
     {
         /// <summary>
+        /// The sign-in audience was not specified
```

Comment:
What does this mean? How will the sign in behave?

Reply:
@MarkZuber let's discuss this point

Reply:
None -- this means one wasn't specified, and since we're validating the entire underlying structure of the ApplicationOptions there won't be confusion in the code around "we specified B2C, but some other code is looking at authority audience only and making a decision"

---
In reply to: [248207428](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/790#discussion_r248207428) [](ancestors = 248207428)